pandas/tests/groupby/test_transform.py

```diff
@@ -958,6 +958,21 @@ def test_groupby_transform_timezone_column(func):
     tm.assert_frame_equal(result, expected)
 
 
+def test_quantile_validation():
+    # GH#27470
+    df = pd.DataFrame(dict(a=[0, 0, 0, 1, 1, 1]))
+    g = df.groupby(np.repeat([0, 1], 3))
+    result = g.quantile(0.5)
+    expected = DataFrame(dict(a=[0.0, 1.0]))
+    tm.assert_frame_equal(result, expected)
```

Comment:
Is this not tested elsewhere?

Reply:
If I eliminate all the reductions from the whitelist in #27470 (which should break everything), only 3 of them had coverage.